src/main/java/de/tum/in/www1/artemis/service/compass/assessment/Assessment.java

```diff
@@ -52,18 +70,8 @@ public void addScore(Score score, Context context) {
         double maxCount = counting.entrySet().stream().mapToInt(Map.Entry::getValue).max().orElse(0);
 
         // calculate the mean amount of points
-        credits /= scoreList.size();
-
-        // calculate the confidence for this value
-        double confidence = maxCount / scoreList.size();
-
-        contextScoreMapping.put(context, new Score(credits, new ArrayList<>(comments), confidence));
-    }
-
-    /**
-     * Used for statistic
-     */
-    public Map<Context, List<Score>> getContextScoreList() {
-        return this.contextScoreList;
+        credits /= scores.size();
```

Comment:
please use a new variable for the mean. It's confusing that the same variable is used for the sum and the mean